README.md

```diff
@@ -1238,6 +1238,7 @@ _See also [Monitoring](#monitoring)_
 - [Baby Buddy](https://github.com/cdubz/babybuddy) - Helps caregivers track baby sleep, feedings, diaper changes, and tummy time. ([Demo](https://babybuddy.herokuapp.com/)) `BSD-2-Clause` `Python`
 - [Dashboard](https://github.com/phntxx/dashboard) - Minimalist homepage for organizing your web applications and bookmarks using JSON-files. `MIT` `Nodejs/Docker`
 - [DashMachine](https://github.com/rmountjoy92/DashMachine) - Another web application bookmark dashboard, with fun features. `GPL-3.0` `Python`
+- [Dashy](https://github.com/lissy93/dashy) - Feature-rich homepage for your homelab, with easy YAML configuration. ([Demo](https://dashy-demo-2.as93.net)) `MIT` `Javascript/Docker`
```

Comment:
> Deploying
> Install dependencies: yarn
>Build: yarn build
>Run: yarn start
>
> Deploying with Docker

`Nodejs/Docker`

Javascript is for static javascript apps you can install without a build system